src/app/examples/floating-action-button-example/floating-action-button-example.component.tns.html

```diff
@@ -1,4 +1,4 @@
-<WrapLayout>
+<StackLayout orientation="vertical">
```

Comment:
Why??

Reply:
forgot to revert it, but I will now also add a marginTop of 10, as we cannot see the top of the buttons otherwise